public/app/features/dashboard/components/AddPanelWidget/AddPanelWidget.tsx

```diff
@@ -261,43 +261,28 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
       display: flex;
       flex-direction: column;
       row-gap: ${theme.spacing.sm};
+      padding: 0 ${theme.spacing.sm} ${theme.spacing.sm} ${theme.spacing.sm};
       height: 100%;
-      margin-left: ${theme.spacing.xl};
-      margin-right: ${theme.spacing.xl};
-      margin-top: ${theme.spacing.base * 4}px;
-      margin-bottom: ${theme.spacing.base * 5}px;
```

Comment:
@kaydelaney tip, try to avoid margins. Padding is almost always better (with some exceptions). Mainly due to things like margin collapse (https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing)